rafay/resource_cloud_credential.go

```diff
@@ -201,14 +206,15 @@ func resourceCloudCredentialCreate(ctx context.Context, d *schema.ResourceData,
 				d.Get("clientsecret").(string),
 				d.Get("subscriptionid").(string),
 				d.Get("tenantid").(string),
-				3)
+				credType)
 			if err != nil {
 				log.Printf("Error while create azure credentials  %s", err.Error())
 				return diag.FromErr(err)
 			}
 			d.SetId(s.ID)
 		} else if d.Get("providertype").(string) == "MINIO" {
 			if d.Get("awscredtype").(string) == "rolearn" {
+				// Discuss this path with @thirumal-rafay
```

Comment:
What to discuss here? @mayank-rafay

Reply:
erroring out now for this case and commented out the terraform implementation code.